test/expect/caffe_mnist.expect

```diff
@@ -0,0 +1,892 @@
+node {
+  name: "conv1/XavierFill"
```

Comment:
Do you still need this file? If so, can you rename to `test/expect/TestTensorBoard.test_mnist.expect` or something?

Reply:
Let's make it a separate PR.